README.md

```diff
@@ -1,7 +1,491 @@
-Manifold Frontend
-====
+Manifold is a high level language for creating all kinds of systems the way we
+write software.
+
+Currently, it focuses on hardware and microfludics design.
+
+# Introduction
+
+In much the same way that computer programs are written in programming
+languages, digital circuits is "written" in a **hardware description language**.
+
+The two hardware languages in wide use today are **VHDL** and **Verilog**. Both
+were designed in the early 1980's and both have remained frozen in time since,
+while our understanding of how to build expressive, powerful, and usable
+programming languages has grown considerably. These languages are in widespread
+use because they are the best tool for the job but they aren't nearly as
+expressive or approachable as modern programming languages.
+
+More recently, several projects have spring up that provide tooling to use
+existing *programming* languages as hardware languages. These projects include
+**[CλaSH](http:#clash.ewi.utwente.nl/ClaSH/Home.html)**,
```

Comment:
`http:#` - the links don't work in Github flavoured markdown. Should these links be `http://` ?